src/tests/ftest/deployment/io_sys_admin.py

```diff
@@ -76,7 +76,7 @@ def test_io_sys_admin(self):
         self.container[-1].create_snap()
         # overwrite the last ior file
         self.ior_cmd.signature.update('456')
-        self.run_ior_with_pool(create_pool=False, create_cont=False)
+        self.run_ior_with_flags(self.ior_np, self.ior_ppn, create_pool=False, create_cont=False)
```

Comment:
I don't think we should add more functions like this to ior_test_base. We're actually trying to move in the opposite direction since the test base is very crowded/messy. I think it would be better to do:
```suggestion
        self.np = self.ior_np
        self.ppn = self.ior_ppn
        self.run_ior_with_pool(create_pool=False, create_cont=False)
```

Reply:
- [ ] Remove useless function